airtouch5py/airtouch5_simple_client.py

```diff
@@ -46,9 +47,21 @@ class Airtouch5SimpleClient:
     _client: Airtouch5Client
     _connection_task: asyncio.Task[None] | None
 
-    def __init__(self, ip: str):
-        self.ip = ip
-        self._client = Airtouch5Client(ip)
+    def __init__(self, ip_or_device):
+
+        if isinstance(ip_or_device, AirtouchDevice):
+            self.device = ip_or_device
+            self.ip = ip_or_device.ip
+
+        # If an IP address string was provided
+        elif isinstance(ip_or_device, str):
```

Comment:
If an ip is provided, could we send a udp discovery packet to it and populate `self.device` from the response?
Then we'll always have the device properties (which I expect we'll need when we want to use them in home assistant)